apps/server/src/modules/alert/alert.config.ts

```diff
@@ -0,0 +1,19 @@
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+import { SchulcloudTheme } from '../server/types/schulcloud-theme.enum';
```

Comment:
If we need it on both places that the schulcloud-theme.enum must be moved to shared/domain.